packages/currency/src/erc20/chains/fantom.ts

```diff
@@ -1,4 +1,4 @@
-import { TokenMap } from './types';
+import { TokenMap } from '../../types';
 
 export const supportedFantomTokens: TokenMap = {
```

Comment:
Consider renaming `supportedFantomERC20` for consistency

Reply:
fixed in e52eb104